data/Kabit Savaiye Bhai Gurdas Ji/235.json

```
@@ -64,7 +64,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Seva Singh": "jYsy mnu ikriq ibriq mY mgnu hoie; swDsMg kIrqn mY n TihrwveI ["
+          "Seva Singh": "jYso mnu ikriq ibriq mY mgnu hoie, swD sMg kIrqn mY n TihrwveI ]"
```

```suggestion
          "Seva Singh": "jYso mnu ikriq ibriq mY mgn hoie; swD sMig kIrqn mY n TihrwveI ]"
```

Same as above for vishram. Also fixing `mgnu` to `mgn` to match source. And `sMg` to `sMig` to match source.